.circleci/config.yml

```
@@ -529,7 +529,8 @@ jobs:
               rm cmake.tar.gz
             cmake --version
             # Install pika dependencies
-            sudo apt install -y \
+            sudo apt install --allow-downgrades \
```

We might as well sneak this in:
```suggestion
            sudo apt update
            sudo apt install --allow-downgrades \
```